src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -1302,31 +1340,31 @@ class DevTask extends AbstractFeatureTask {
             }
         }
 
-        if (dockerfile == null) {
-            File buildDockerfileSetting = project.liberty.dev.dockerfile; // get from build.gradle
-            if (buildDockerfileSetting != null) {
-                setDockerfile(buildDockerfileSetting.getAbsolutePath()); // setDockerfile will convert it to canonical path
+        if (containerfile == null) {
+            File buildcontainerfileSetting = project.liberty.dev.containerfile; // get from build.gradle
```

Comment:
Do we also need to check for the alias in build.gradle if null is returned?

Reply:
So I think at this point the `containerfile` would have been set if `dockerfile` were specified. What I meant is do we need to check for `project.liberty.dev.dockerfile` if there is not `project.liberty.dev.containerfile`? This comment applies to all the other attributes too.